app/controllers/timer_sessions_controller.rb

```diff
@@ -16,7 +16,7 @@ def index
                       .sort_by(&:start_time)
                       .reverse
                       .group_by(&:entry_date)
-    @timer_offset = offset_for_time_zone(@current_user)
+    @timer_offset = TimerSessionsController.offset_for_time_zone(@current_user)
```

Comment:
Is `@timer_offset` in hours? If so, this could be problematic for some asian timezones. There are half-hour delays, e.g. https://de.wikipedia.org/wiki/UTC%2B5:30.

* Either rename it to `@timer_offset_hours` and handle integer division on line 99 properly
* Or keep it in seconds.

Reply:
Done